### PR TITLE
[NOJIRA]Eliminate duplicated build ids

### PR DESCRIPTION
**Pull request by sylank**
Fixes: #10

### CODE REVIEW

File: build_id_getter.go [hunk 1]
```diff
@@ -21,16 +21,16 @@ func NewBuildIDGetter(finishedStages model.FinishedStages, targetNames []string)
 }
 
 func (bg BuildIDGetter) GetBuildIDs() ([]string, error) {
-	var buildIDs []string
```
Comment (imrekel) [inline build_id_getter.go:24]: I think you could cover this with the tests easily just by adding a duplicate build id into the input data

**Review by imrekel — APPROVED** [review]
